vsts/pipelines/templates/_securityChecks.yml

```diff
@@ -15,6 +15,7 @@ steps:
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
   displayName: 'Run CredScan'
   inputs:
+    toolMajorVersion: V2
```

Comment:
this is to get rid of following warning in security stage of our builds

> Enabling TLS 1.2...
> ##[warning]This build is running CredScan v1, which is on path to deprecation in the coming months. Please update this build definition to run CredScan v2, by selecting 'CredScan V2' in the 'Tool Major Version' dropdown in the build task UI (toolMajorVersion: V2 in yaml). You can learn more about CredScan v2, which was released in October of 2019, at https://aka.ms/CredScan. 
> Initializing analysis environment...
>

Reply:
Seems this CredScan has some issue with this upgrade version, I saw you're contacting the analysis support team, thanks, let's see how it can be solved.